src/components/VerticalListMenu/VerticalListMenu.spec.jsx

```diff
@@ -0,0 +1,160 @@
+						<VerticalListMenu.Item />
+					</VerticalListMenu>
+				</VerticalListMenu.Item>
+			</VerticalListMenu>
+		);
+
+		assert.equal(wrapper.find('.lucid-VerticalListMenu-Item-nested-list').children().length, 1);
+	});
+
+	it('should favor child props over parent level indicies', () => {
+		const wrapper = shallow(
+			<VerticalListMenu selectedIndices={[0]} expandedIndices={[2]}>
+				<VerticalListMenu.Item>
+					Selected one
+				</VerticalListMenu.Item>
+
+				<VerticalListMenu.Item isSelected={true}>
+					Selected two
+				</VerticalListMenu.Item>
+
+				<VerticalListMenu.Item>
+					Expanded one
+				</VerticalListMenu.Item>
+
+				<VerticalListMenu.Item isExpanded={true}>
+					Expanded two
+				</VerticalListMenu.Item>
+			</VerticalListMenu>
+		);
+
+		assert.equal(wrapper.find('.lucid-VerticalListMenu-Item-content-is-selected').length, 2);
+		assert.equal(wrapper.find('.lucid-VerticalListMenu-Item-nested-list-is-expanded').length, 2);
+	});
+
+	it('should fire onSelect', () => {
+		const onSelect = sinon.spy();
+		const wrapper = shallow(
+			<VerticalListMenu onSelect={onSelect}>
+				<VerticalListMenu.Item>One</VerticalListMenu.Item>
+				<VerticalListMenu.Item>Two</VerticalListMenu.Item>
+			</VerticalListMenu>
+		);
+
+		wrapper.find('.lucid-VerticalListMenu-Item-content').at(1).simulate('click');
+
+		assert(onSelect.called);
+		assert.equal(onSelect.args[0][0], 1, 'wrong index on the onSelect callback');
+		assert(_.has(onSelect.args[0][1], 'event'), 'missing `event` on the onSelect callback');
+		assert(_.has(onSelect.args[0][1], 'props'), 'missing `props` on the onSelect callback');
+	});
+
+	it('should fire onSelect on the child', () => {
+		const onSelect = sinon.spy();
+		const wrapper = shallow(
+			<VerticalListMenu>
+				<VerticalListMenu.Item>One</VerticalListMenu.Item>
+				<VerticalListMenu.Item onSelect={onSelect}>Two</VerticalListMenu.Item>
+			</VerticalListMenu>
+		);
+
+		wrapper.find('.lucid-VerticalListMenu-Item-content').at(1).simulate('click');
+
+		assert(onSelect.called);
+		assert.equal(onSelect.args[0][0], 1, 'wrong index on the onSelect callback');
+		assert(_.has(onSelect.args[0][1], 'event'), 'missing `event` on the onSelect callback');
+		assert(_.has(onSelect.args[0][1], 'props'), 'missing `props` on the onSelect callback');
+	});
+
+	it('should show expanders based on hasExpander', () => {
+
+		const wrapper = shallow(
+			<VerticalListMenu>
+				<VerticalListMenu.Item />
+				<VerticalListMenu.Item hasExpander={true} />
+				<VerticalListMenu.Item hasExpander={true} />
+			</VerticalListMenu>
+		);
+
+		assert.equal(wrapper.find('.lucid-VerticalListMenu-Item-expander').length, 2);
+	});
+});
+
+describeWithDOM('VerticalListMenu', () => {
+	// Since onToggle uses a Button under the hood, we need to do a `mount` test
+	// to make sure it's passing the `event` object around correctly
+	it('should fire onToggle', () => {
+		const onToggle = sinon.spy();
+		const wrapper = mount(
```

Comment:
It's a good idea to call `wrapper.unmount` _after each_ test just in case the component does any cleanup on `componentWillUnmount`. Even if it doesn't do anything, it's a good habit to get into.